parm/use_cases/model_applications/s2s/UserScript_obsERA_obsOnly_Stratosphere/meridonial_mean.yaml

```diff
@@ -0,0 +1,2 @@
+input_filename: 
+- !ENV '${INPUT_FILE_NAME}/model_applications/s2s/UserScript_obsERA_obsOnly_Stratosphere/SSWC_v1.0_varFull_ERAi_d20130106_s20121107_e20130307_c20160701.nc'
```

Comment:
The input for the use case needs to be added to the directory with the same name as this feature branch on the DTC web server to make it available in the automated tests. Detailed steps to add new data is here: https://metplus.readthedocs.io/en/develop/Contributors_Guide/add_use_case.html#input-data
**Be sure to read each step carefully! Some instructions are only run on certain conditions.**